### DISCUSSION TITLE
Is live streaming implemented

### DISCUSSION THREAD

**Discussion by armeya25** [yuankunzhang/charming#43] [Q&A]
Can I plot streaming candlestick data from brocker,
If yes,
Plz provide any example if possible.
Thanks in advance

**Comment by kapteinkaald** [comment]
Hi, what do you mean by this? I do not understand what you want to do.

**Comment by armeya25** [comment]
I want to plot data I get from brocker/ websockets, say every 1 minutes,
Not a static data from .CSV or local files

**Reply by kapteinkaald** [reply]
OK! Me too. For me its the graph charts I want to plot. Would be a good addition, but I am currently not able to implement this myself.

**Reply by armeya25** [reply]
Can't you push data to vecs and redraw,
It may be expensive but may work

**Reply by kapteinkaald** [reply]
hmm. Quite new to rust so I have no idea. But I will try! Thanks!

**Reply by armeya25** [reply]
I am just starting to understand rust so no can't help you there,
Also you may look into "lightweight charts" library there you may find answer for live plotting

**Reply by armeya25** [reply]
Discord link is dead
